### PR TITLE
add linaro-forge uenv check

### PR DESCRIPTION
**Pull request by jgphpc**
https://jira.cscs.ch/browse/VCUE-284
Tested with linaro-forge/24.0:1310006277

### CODE REVIEW

File: checks/tools/profiling_and_debugging/linaro-forge.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class linaro_ddt(rfm.RunOnlyRegressionTest):
+    valid_systems = ['*']
+    valid_prog_environs = ['+ddt']
+    sourcesdir = None
```
Comment (victorusu) [inline checks/tools/profiling_and_debugging/linaro-forge.py:10]: this is not necessary

File: checks/tools/profiling_and_debugging/linaro-forge.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        """
+        Sanity checks:
+         - "Error communicating with Licence Server velan.cscs.ch: The proxy
+         type is invalid for this operation"
+         - Offline log written to: 'rpt.txt'
+         - message (0): Startup complete.
+         - message (n/a): Every process in your program has terminated.
+        """
+        return sn.all([
+            sn.assert_not_found(r'Error communicating with Licence Server',
```
Comment (victorusu) [inline checks/tools/profiling_and_debugging/linaro-forge.py:41]: I do not think you need the `r` in these strings

File: checks/tools/profiling_and_debugging/linaro-forge.py (hunk 1 above)
Comment (jgphpc) [reply checks/tools/profiling_and_debugging/linaro-forge.py:10]: line 10 ? yes it is: I do not want/need to copy the src dir.

File: checks/tools/profiling_and_debugging/linaro-forge.py (hunk 2 above)
Comment (jgphpc) [reply checks/tools/profiling_and_debugging/linaro-forge.py]: https://github.com/eth-cscs/cscs-reframe-tests/pull/201/commits/3c59c4796f83784a621bab5afafaadf34696b8af

**Review by teojgo — CHANGES_REQUESTED** [review]
Add also the copyright header

**Review by teojgo — APPROVED** [review]

**Pull request merged**